test/memspaces/memspace_fixtures.hpp

```diff
@@ -51,7 +51,7 @@ struct numaNodesTest : ::umf_test::test {
     unsigned long maxNodeId = 0;
 };
 
-using isQuerySupportedFunc = bool (*)(size_t);
+using isQuerySupportedFunc = void (*)(size_t);
```

Comment:
can you describe in a commit message, in a sentence or two, why this change from void to bool. This seem like the most important part of this PR (it changes very little of UT_ASSERTs, as the title would suggest 😉 )

Reply:
Something like the first paragraph of the PR description? Without the sentence with GTEST docs.